dotcom-rendering/src/components/TagFrontHeader.tsx

```diff
@@ -0,0 +1,322 @@
+	${from.desktop} {
+		grid-template-columns:
+			[viewport-start] minmax(0, 1fr)
+			[content-start title-start]
+			repeat(6, 60px) [content-end]
+			repeat(4, 60px)
+			[title-end image-start]
+			repeat(2, 60px)
+			[image-end]
+			minmax(0, 1fr) [viewport-end];
+	}
+
+	${from.leftCol} {
+		grid-template-rows:
+			[headline-start content-start image-start] auto
+			[headline-end content-end image-end] auto;
+
+		grid-template-columns:
+			[viewport-start] minmax(0, 1fr)
+			[title-start]
+			repeat(2, 60px)
+			[title-end content-start]
+			repeat(7, 60px) [content-end]
+			repeat(3, 60px)
+			[image-start]
+			repeat(2, 60px)
+			[image-end]
+			minmax(0, 1fr) [viewport-end];
+	}
+
+	${from.wide} {
+		grid-template-columns:
+			[viewport-start] minmax(0, 1fr)
+			[title-start]
+			repeat(3, 60px)
+			[title-end content-start]
+			repeat(8, 60px) [content-end]
+			repeat(2, 60px)
+			[ image-start]
+			repeat(2, 60px)
+			[image-end]
+			repeat(1, 60px)
+			minmax(0, 1fr) [viewport-end];
+	}
+`;
+
+const sectionHeadline = css`
+	grid-row: headline;
+	grid-column: title;
+
+	display: flex;
+	flex-direction: column;
+
+	${from.leftCol} {
+		position: relative;
+		::after {
+			content: '';
+			display: block;
+			width: 1px;
+			top: 0;
+			height: 1.875rem;
+			right: -10px;
+			position: absolute;
+			background-color: ${palette.neutral[86]};
+		}
+	}
+`;
+
+const paddings = css`
+	padding-top: ${space[2]}px;
+	padding-bottom: ${space[2]}px;
+`;
+
+const sectionImage = css`
+	grid-row: image;
+	grid-column: image;
+	justify-self: end;
+`;
+
+const sectionContent = css`
+	margin: 0;
+
+	grid-column: content;
+	grid-row: content;
+`;
+
+/** element which contains border and inner background colour, if set */
+const decoration = css`
+	grid-row: 1 / -1;
+	grid-column: 1 / -1;
+
+	${from.tablet} {
+		grid-column: 2 / -2;
+	}
+
+	border-width: 1px;
+	border-color: ${palette.neutral[86]};
+	border-style: none;
+`;
+
+/** only visible once content stops sticking to left and right edges */
+const sideBorders = css`
+	${from.tablet} {
+		margin: 0 -20px;
+		border-left-style: solid;
+		border-right-style: solid;
+	}
+`;
+
+const titleStyle = css`
+	${until.leftCol} {
+		max-width: 74%;
+	}
+`;
+
+const paragraphStyle = css`
+	${headline.xxxsmall()}
+	color: ${palette.neutral[46]};
+`;
+
+const buildElementTree = (node: Node): ReactNode => {
+	if (isElement(node)) {
+		switch (node.nodeName) {
+			case 'A':
+				return jsx('a', {
+					href: node.attributes.getNamedItem('href')?.value,
+					target: node.attributes.getNamedItem('target')?.value,
+					children: Array.from(node.childNodes).map(buildElementTree),
+				});
+			default:
+				return jsx(node.tagName.toLowerCase(), {
+					children: Array.from(node.childNodes).map(buildElementTree),
+				});
+		}
+	} else if (node.nodeType === node.TEXT_NODE) {
+		return node.textContent;
+	} else {
+		logger.warn('TagFrontHeader: Unknown element received', {
+			isDev: process.env.NODE_ENV !== 'production',
```

Comment:
Wondering what the benefit to this field is 🤔 I know its used by other cases of `buildElementTree` so I'm happy for it to stay for consistency sake